test/upgradeableProxy/Dispatcher.upgrade.t.sol

```diff
@@ -106,41 +149,70 @@ contract ChannelHandShakeUpgradeUtil is ChannelHandshakeUtils {
         bytes32 slot1 = keccak256(abi.encode(portAddress, nextSequenceRecvSlot));
         slot = keccak256(abi.encode(channelId, slot1));
     }
+
+    function mockHandShakeProofs(LocalEnd memory _local, ChannelEnd memory _remote) public {
+        // Mock the proofs required to initialize a channel
+        mockProofs(ChannelState.TRY_PENDING, _remote.portId, _local.portId, _remote.channelId, _local.channelId);
+        mockProofs(ChannelState.ACK_PENDING, _local.portId, _remote.portId, _local.channelId, _remote.channelId);
+        mockProofs(ChannelState.CONFIRM_PENDING, _remote.portId, _local.portId, _remote.channelId, _local.channelId);
+    }
```

Comment:
why do you need to mock proofs? can't you use a dummy verifier in this case? the proof verifier holds no state so it shouldn't matter too much in the upgrade path right?

Reply:
should be updated to only use dummy client now